samples/ObjectTracker/README.md

```diff
@@ -0,0 +1,153 @@
+
+```csharp
+Windows.Foundation.Rect boundingRect = binding.BoundingRect;
+bool trackerSucceeded = binding.Succeeded;
+```
+
+### Sample app code walkthrough
+
+The core skill initialization logic is in `InitializeSkillAsync`:
+
+```csharp
+private async Task InitializeSkillAsync(ISkillExecutionDevice executionDevice = null)
+{
+    m_descriptor = new ObjectTrackerDescriptor();
+    m_availableExecutionDevices = await m_descriptor.GetSupportedExecutionDevicesAsync();
+    if (m_availableExecutionDevices.Count == 0)
+    {
+        NotifyUser("No execution devices available, this skill cannot run on this device", NotifyType.ErrorMessage);
+        return; // Abort
+    }
+
+    // Either create skill using provided execution device or let skill create with default device if none provided
+    if (executionDevice != null)
+    {
+        m_skill = await m_descriptor.CreateSkillAsync(executionDevice) as ObjectTrackerSkill;
+    }
+    else
+    {
+        m_skill = await m_descriptor.CreateSkillAsync() as ObjectTrackerSkill;
+    }
+
+    m_bindings = new List<ObjectTrackerBinding>();
+    m_trackerHistory = new List<List<TrackerResult>>();
+}
+```
+
+Note that we have a `List<ObjectTrackerBinding>` instead of a single binding. This is an example of how a single `ObjectTrackerSkill` instance can be used to track multiple objects, with each `ObjectTrackerBinding` corresponding to a single tracked object.
+
+The core skill evaluation logic, including initialization logic for new trackers, is in `RunSkillAsync`:
+
+```csharp
+private async Task RunSkillAsync(VideoFrame frame)
+{
+    /* snip */
+    for (int i = 0; i < m_bindings.Count; i++)
+    {
+        await m_bindings[i].SetInputImageAsync(frame);
+        m_skillEvalStopWatch.Restart();
+        await m_skill.EvaluateAsync(m_bindings[i]);
+        m_skillEvalStopWatch.Stop();
+        evalTicks += m_skillEvalStopWatch.ElapsedTicks;
+
+        // Add result to history
+        m_trackerHistory[i].Add(
+            new TrackerResult()
+            {
+                boundingRect = m_bindings[i].BoundingRect,
+                succeeded = m_bindings[i].Succeeded
+            }
+        );
+    }
+    /* snip */
+    for (int i = 0; i < m_drawnRects.Count; i++)
+    {
+        ObjectTrackerBinding binding = await m_skill.CreateSkillBindingAsync() as ObjectTrackerBinding;
+        await m_skill.InitializeTrackerAsync(binding, frame, m_drawnRects[i]);
+        m_bindings.Add(binding);
+
+        // Add corresponding tracker history
+        m_trackerHistory.Add(
+            new List<TrackerResult> {
+                        new TrackerResult()
+                        {
+                            boundingRect = binding.BoundingRect,
+                            succeeded = true
+                        }
+            }
+        );
+    }
+    m_drawnRects.Clear();
+    /* snip */
```

Comment:
this block of code needs a come comments or header that dissociate it from the eval logic in RunSkillAsync (i.e. // Initialize any new trackers selected by the user